DSCResources/MSFT_xCluster/MSFT_xCluster.psm1

```diff
@@ -104,6 +104,8 @@ function Set-TargetResource
 
             New-Cluster -Name $Name -Node $env:COMPUTERNAME -StaticAddress $StaticIPAddress -NoStorage -Force
 
+            Start-Sleep -Seconds 60
```

Comment:
@claudiospizzi - This discussion is still unresolved. Can you please update the code to not just rely on sleep alone?